source/process/release-process.md

```diff
@@ -268,23 +284,33 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
 
-1. Build:  
+1. Dev:
```

Comment:
Let's 
1) move dev items below `Tag a new release (e.g. 1.1.1) and run an official build` in the Build section
2) rename 'build' to 'dev'

Reply:
Done